### PR TITLE
feat(l10n): update Ukrainian translations

### PR DESCRIPTION
**Pull request by sakhnik**

### CODE REVIEW

**Comment by clason** [comment]
@echasnovski

File: src/nvim/po/uk.po [hunk 1]
```diff
@@ -4624,6 +4636,10 @@ msgstr "E227: Вже є заміна клавіш для %s"
 msgid "E460: Entries missing in mapset() dict argument"
 msgstr "E460: Бракує записів у словниковому аргументі mapset()"
 
+#, c-format
+msgid "E1276: Illegal map mode string: '%s'"
+msgstr "E1276: Недозволений режим відображення: '%s'"
```
Comment (echasnovski) [inline src/nvim/po/uk.po:4641]: The translation means more along the lines of "Illegal display mode". I'd say "Недозволений режим заміни клавіш" is closer to the original (according to the rest of the translation file).

**Comment by echasnovski** [comment]
Overall looks good to me. There is one place which seems somewhat strange, but if this comes directly from Vim (and is already merged), I'd say this is not a huge deal.

File: src/nvim/po/uk.po (hunk 1 above)
Comment (sakhnik) [reply src/nvim/po/uk.po:4641]: Thanks, I'll fix up!

File: src/nvim/po/uk.po [hunk 2]
```diff
@@ -5764,6 +5785,110 @@ msgstr "E554: Синтаксична помилка в %s{...}"
...
+msgstr "Не вдалося відкрити тимчасовий файл для запису, показую в stderr…"
+
+msgid "E874: (NFA) Could not pop the stack!"
+msgstr "E874: (NFA) Не вдалося витягнути зі стеку!"
+
+msgid ""
+"E875: (NFA regexp) (While converting from postfix to NFA),too many states "
+"left on stack"
+msgstr ""
+"E875: (NFA regexp) (При перетворенні з постфікс у NFA), загабато станів "
```
Comment (echasnovski) [inline src/nvim/po/uk.po:5886]: @sakhnik, sorry only on second proof read found this typo: "загабато" -> "забагато"

Comment (sakhnik) [reply src/nvim/po/uk.po]: Thanks, fixing up.

**Comment by echasnovski** [comment]
@clason, as of now this looks good to merge.

**Pull request merged**